CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## 0.8.0
 > Released on 2025/??
 
+### BREAKING CHANGES
+* `service` field in `konnect_api_implementation` resource is now nested under `service_reference` field.
+* `spec` field in `konnect_api_version` resource is now required
```

Comment:
`spec` being required is technically a breaking change, even though API was erroring out earlier if absent ([source](https://github.com/Kong/platform-api/pull/1556)) - so I've added.